src/CardInfo.js

```diff
@@ -0,0 +1,47 @@
+import { useParams, Link } from "react-router-dom"
+import axios from "axios"
+import { useState, useEffect } from "react"
+
+const CardInfo = () => {
+
+    const [cardData, setCardData] = useState({})
+    const params = useParams()
+    const urlParamsValue = (params.id)
+
+    useEffect(() => {
+		axios({
+			url: `https://api.magicthegathering.io/v1/cards/${urlParamsValue}`,
+			
+		}).then((allMagicCards) => {
+			setCardData(allMagicCards.data.card)
+			console.log(allMagicCards.data.card)
+		})
+	},[urlParamsValue])
+
+    const { name, imageUrl, manaCost, colors, type, rarity, setName, text, power, toughness, flavor } = cardData
+
+    return (
+        <div className="cardContainer">
+            <div className="cardInfoContainer">
+                <h2>{name}</h2>
+                <p>Type: {type}</p>
```

Comment:
you can try to have span's on the type, cmc, colours, etc so you can style them differently (bold, or change size) from the received data